crates/host/src/wasmbus/mod.rs

```diff
@@ -1554,6 +1668,8 @@ impl Host {
                     calls: calls_abort,
                     runtime: self.runtime.clone(),
                     handler: handler.clone(),
+                    claims: claims.clone(),
```

Comment:
Do you need to clone before _and_ after the async move?

Reply:
Yep! Unfortunate